src/components/grids/TabbedDisplay/index.tsx

```diff
@@ -77,25 +73,11 @@ export type TabbedDisplayProps = {
 export default function TabbedDisplay({
   tabs,
   activeTab,
+  onClick,
   styleOverrides = {},
   themeRole,
 }: TabbedDisplayProps) {
-  const [activeTabInternal, setActiveTabInternal] = useState(
-    activeTab ?? tabs[0].displayName
-  );
-
-  // Listen for changes to the `activeTab` prop. This allows the component
-  // to be controlled programmatically.
-  useEffect(() => {
-    if (activeTab && activeTab !== activeTabInternal) {
-      const matchingTabRecord = tabs.find(
-        (tab) => tab.displayName === activeTab
-      );
-      matchingTabRecord && setActiveTabInternal(activeTab);
-      matchingTabRecord?.onSelect && matchingTabRecord.onSelect();
-    }
-  }, [activeTab]);
-
+  const tabContent = tabs.find(tab => tab.displayName === activeTab)!.content
```

Comment:
I'd remove the [non-null assertion](https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator), because it's possible that the user's passed `activeTab` will not match the display names of any of the provided tabs.

```suggestion
  const tabContent = tabs.find(tab => tab.displayName === activeTab).content
```

Reply:
In the future, we might also consider:

1. Making `activeTab` optional
2. Providing some sort of `unselectedTabPanel: ReactNode` prop, which would let us provide a custom view for the case where the `activeTab` isn't provided or doesn't match any of passed `tabs`